src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -78,6 +78,7 @@ export interface RegistrationFilingIF {
     effectiveDate?: string // not saved by UI but may be provided by API
     filingId?: number // not saved by UI but may be provided by API
     folioNumber?: string // optional and only displayed for certain account types
+    isTransactionalFolioNumber?: boolean // Optional - applies to registration Premium accounts only
```

Comment:
I am not sure if the legal api supports this, from my local tests it seems to. I looked through the legal api and I couldn't tell if it did or not.

Reply:
The Legal API will save the filing JSON (ie, this object) verbatim in the filings table.

Whether the Filer looks at this property and does something special with it, I don't know. It could be tested by setting a TFN, filing a registration, and then looking at the receipt. (And subsequently filing a change with no TFN and confirming this original TFN is not used either, ie, it really was for the registration only.)

Reply:
So the legal API only acts to save the JSON and doesn't actually care what is the contents of the object are?

Reply:
The Legal API only cares about the objects only when the filing is filed (draft = false), at which point it does schema and some custom validation.